src/afl-cc.c

```diff
@@ -914,6 +918,13 @@ static void edit_params(u32 argc, char **argv, char **envp) {
 
   }
 
+  if (getenv("AFL_USE_LSAN")) {
+    cc_params[cc_par_cnt++] = "-includesanitizer/lsan_interface.h";
+  }
+
+  cc_params[cc_par_cnt++] =
+      "-D__AFL_LEAK_CHECK()=__lsan_do_leak_check()";
+
```

Comment:
I now see that -D__AFL_LEAK_CHECK  is not part of if(AFL_USE_ASAN) above. IMHO it makes more sense to move it in there?

Reply:
I had originally put it here (forgetting to put if(..)) to collate all the definitions in one place, but I suppose it will fit fine in the if(..) above